src/errors.ts

```
@@ -387,13 +406,6 @@ export const NO_RESPONSE_MESSAGE =
  * @see enrichBusinessRequestError
  */
 function selectServerErrorMessage({ response }: AxiosError): string | null {
-  // For examples of DRF errors, see the pixiebrix-app repository:
```

Should this method ever be called if response is null? If not, we should throw an error here if response is null?

Or, if we're relying on `safeGuessStatusText(response.status)` to return null, we should document this expectation